src/main/java/org/primefaces/component/schedule/ScheduleRenderer.java

```diff
@@ -118,7 +119,7 @@ protected void encodeEventsAsJSON(FacesContext context, Schedule schedule, Sched
                 jsonObject.put("className", event.getStyleClass());
                 jsonObject.put("description", event.getDescription());
                 jsonObject.put("url", event.getUrl());
-                jsonObject.put("rendering", event.getRenderingMode());
+                jsonObject.put("rendering", Objects.toString(event.getRenderingMode(), null));
```

Comment:
I see you are doing Objects.toString but not .toLowerCase()?

Reply:
That because in PrimeFaces it is an enum following java naming conventions. The ScheduleRenderingMode enum have internally the correct string for the full calendar. 
.toLowerCase() would not work for ScheduleRenderingMode.INVERSE_BACKGROUND because it must resolve to 'inverse-background'.
You must call getRendering() or toString() on the enum. To be null save I use Objects.toString()

The behavior of org.json changed. In the old version it did a .toString() on all untested object types. Now it tests for enums and calls .name() on them. So now longer the internal string is used but the name.